src/halide_ir.fbs

```diff
@@ -73,6 +73,12 @@ enum ForType: byte {
     GPULane,
 }
 
+enum LoopPartitionPolicy: byte {
```

Comment:
@TH3CHARLie when we change the serialization format like this, do we need to bump a version number somewhere?

Reply:
we don't need to bump the version now, once https://github.com/halide/Halide/pull/7905 is merged, we do need to bump the serialization format version